Documentation/data-collection.md

```diff
@@ -333,6 +333,21 @@ data:
     # (@red-hat-data-services/odh, Open Data Hub team) The total number of users of RHODS using each component.
     - '{__name__="rhods_total_users"}'
     #
+    # (etcd team) 99th procentile of etcd wal fsync duration.
+    - '{__name__="histogram_quantile_99:etcd_disk_wal_fsync_duration_seconds_bucket:sum"}'
+    #
+    # (etcd team) 90th procentile of etcd wal fsync duration.
+    - '{__name__="histogram_quantile_90:etcd_disk_wal_fsync_duration_seconds_bucket:sum"}'
```

Comment:
what is the cost of adding this 90th percentile? I am not sure its worth it.

Reply:
could we instead include p99 for `etcd_disk_backend_commit_duration_seconds_bucket` ?

Reply:
I thought the 90th showed a good picture when I was running these queries, but happy to replace it with `etcd_disk_backend_commit_duration_seconds_bucket `.